ui/styl/modules/fonts.styl

```diff
@@ -0,0 +1,14 @@
+// partially copied from https://github.com/code42day/stylus-font-face
+// TODO: restrict glyph ranges and inline fonts
+// TODO: possibly include other font types (eg ttf)
+file(name, ext, path)
+  path + '/' + name + '.' + ext
+
+font-face(family, style='normal')
+  @font-face
+    font-family family
+    src url(file(family, 'woff2', 'fonts')) format("woff2"),
+        url(file(family, 'woff', 'fonts')) format("woff")
+    font-weight normal
+    font-style style
+    text-rendering optimizeLegibility
```

Comment:
Missing a newline here.  Scrap the PR, try again, next time try not to embarrass yourself.